### PR TITLE
add more questions to the form, adapted from scientific research paper

### PR DESCRIPTION
**Pull request by fossecode**

### CODE REVIEW

**Comment by fossecode** [comment]
Relates to #245

File: app/util/report-aggregator.test.ts [hunk 1]
```diff
@@ -78,7 +80,8 @@ const reports: CovidReport[] = [
       [Symptom.NO_TASTE]: false,
       [Symptom.NO_SMELL]: false,
       [Symptom.SLIME_COUGH]: false,
-      [Symptom.RUNNY_NOSE]: false
+      [Symptom.RUNNY_NOSE]: false,
```
Comment (adriaandotcom) [inline app/util/report-aggregator.test.ts:83]: Can't read this without thinking of an actual nose running 👃 🏃

**Review by adriaandotcom — APPROVED** [review]
Looks great to me, but still draft so will wait for it to be finished

File: app/util/report-aggregator.test.ts (hunk 1 above)
Comment (fossecode) [reply app/util/report-aggregator.test.ts:83]: 😆

**Comment by fossecode** [comment]
> Looks great to me, but still draft so will wait for it to be finished

Ready now 👍

**Review by adriaandotcom — APPROVED** [review]

**Pull request merged**